zebra-chain/src/block.rs

```diff
@@ -116,6 +119,30 @@ impl Block {
 
         Ok(())
     }
+
+    /// Access the sprout::Nullifiers from all transactions in this block.
```

Comment:
Nit: Documentation link.

```suggestion
    /// Access the [`sprout::Nullifier`]s from all transactions in this block.
```